schema/cidr_array.go

```diff
@@ -22,6 +22,10 @@ func (*CIDRArray) Type() ValueType {
 	return TypeCIDRArray
 }
 
+func (dst *CIDRArray) Size() int {
+	return len(dst.Elements) * 32
```

Comment:
```suggestion
	return len(dst.Elements) * 32
```

Similar to under comments, maybe we can use the underlying type value instead of `32`, i.e. `CIDR.Size()`